graphql_compiler/tests/test_helpers.py

```diff
@@ -83,8 +83,8 @@
 
     directive @recurse(
         \"\"\"
-        Recurse up to this many times on this edge. A depth of 1 produces the current
-        vertex and its immediate neighbors along the given edge.
+        Recurse up to this many times on this edge. A depth of 1 produces the current \
+vertex and its immediate neighbors along the given edge.
```

Comment:
I think that the failed tests might be due to this change. The linter errors are due to isort.

Reply:
That's weird, my local tests refused to pass unless I made this change... 👀

Reply:
Just tried it locally and the test fails for me as well, but reverting it produces another failing test.

Reply:
Maybe you need to delete/escape the whitespace before "Recurse" and after "edge." too?

Reply:
The failing test was unrelated — it was simply producing an alternate but equally-valid order of definitions in the schema autogenerated from OrientDB. I added an order-independent schema definition comparison function and used it. It isn't perfect (let's not add any double newlines in docstrings 😄 ) but it'll be plenty good enough for tests.